src/reader-activation-auth/auth-form.js

```diff
@@ -157,6 +183,12 @@ window.newspackRAS.push( function ( readerActivation ) {
 						body.set( 'reader-activation-auth-form', 1 );
 						body.set( 'npe', emailInput.value );
 						body.set( 'action', 'link' );
+						if ( readerActivation.getCheckoutStatus() ) {
```

Comment:
NIT suggestion: I'd assume `getCheckoutStatus()` would return a string rather than a bool. Maybe this is more of a `hasCheckout()` or `isPendingCheckout()` type of function for a bool return?

Reply:
Great idea. I went with `isPendingCheckout` in https://github.com/Automattic/newspack-plugin/pull/3343/commits/f68ba174858d8213dacda9f734355bd60f9fd638